taca/flowcell_parser/classes.py

```diff
@@ -0,0 +1,322 @@
+
```

Comment:
makes sense, but as it _is_ part of taca, would you mind moving it to `illumina` module?

Maybe implement it as a plugin as Pontus did with the `taca-ngi-pipeline` module?